src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.Desktop.cs

```diff
@@ -29,9 +29,14 @@ internal partial class AnalyzerAssemblyLoader
     {
```

Comment:
Is the doc comment above still up to date? Also it will clash with the other partial implementation, so consider removing it.

Reply:
yeah that is very much out of date.